packages/venia-ui/lib/components/AddressBookPage/addEditDialog.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { bool, func, object, shape } from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+import { isRequired } from '@magento/venia-ui/lib/util/formValidators';
+
+import Checkbox from '@magento/venia-ui/lib/components/Checkbox';
+import Country from '@magento/venia-ui/lib/components/Country';
+import Dialog from '@magento/venia-ui/lib/components/Dialog';
+import Field from '@magento/venia-ui/lib/components/Field';
+import FormError from '@magento/venia-ui/lib/components/FormError';
+import Postcode from '@magento/venia-ui/lib/components/Postcode';
+import Region from '@magento/venia-ui/lib/components/Region';
+import TextInput from '@magento/venia-ui/lib/components/TextInput';
+import defaultClasses from './addEditDialog.css';
+
+const AddEditDialog = props => {
+    const {
+        formErrors,
+        formProps,
+        handleCancel,
+        handleConfirm,
```

Comment:
Should these be `onCancel` and `onConfirm` as prop names?

Reply:
Nice catch, renamed in https://github.com/magento/pwa-studio/pull/2879/commits/8083d2a681f94d963c38306b8dd7a50ecbcc2c91 and unit tests updated in https://github.com/magento/pwa-studio/pull/2879/commits/cbc51f8fed92a17749b70ff6b35108c7d8e30492.